tests/unit/amo/components/TestSearchResult.js

```diff
@@ -144,6 +144,85 @@ describe(__filename, () => {
     expect(root).toHaveClassName('SearchResult--theme');
   });
 
+  it('displays the thumbnail image as the default src for static theme', () => {
+    const headerImageThumb =
+      'https://addons.cdn.mozilla.net/mytestthumb/12345.png';
+    const newPreview = [
+      {
+        ...fakePreview,
+        thumbnail_url: headerImageThumb,
+      },
+    ];
+
+    const addon = createInternalAddon({
+      ...fakeAddon,
+      type: ADDON_TYPE_STATIC_THEME,
+      previews: newPreview,
+    });
+
+    const root = render({ addon });
+    const image = root.find('.SearchResult-icon');
+
+    expect(image.prop('src')).toEqual(headerImageThumb);
+  });
+
+  it('displays srcSet values if preview has multiple options', () => {
+    const headerImageThumb = 'https://addons.cdn.mozilla.net/thumb/12345.png';
+    const headerImageFull = 'https://addons.cdn.mozilla.net/full/54321.png';
+    const thumbWidth = 450;
+    const fullWidth = 900;
+
+    const newPreview = [
+      {
+        ...fakePreview,
+      },
+      {
+        ...fakePreview,
+        image_url: headerImageFull,
+        thumbnail_url: headerImageThumb,
+        image_size: [fullWidth, 200],
+        thumbnail_size: [thumbWidth, 100],
+      },
+    ];
+
+    const addon = createInternalAddon({
+      ...fakeAddon,
+      type: ADDON_TYPE_STATIC_THEME,
+      previews: newPreview,
+    });
+
+    const root = render({ addon });
+    const image = root.find('.SearchResult-icon');
+
+    expect(image.prop('srcSet')).toEqual(
+      `${headerImageThumb} ${thumbWidth}w, ${headerImageFull} ${fullWidth}w`,
+    );
+  });
+
+  // TODO: This can be removed once migration happens.
+  it('displays a fallback image for themes that only have 1 preview option', () => {
+    const headerImageThumb = 'https://addons.cdn.mozilla.net/thumb/1.png';
+
+    const newPreview = [
```

Comment:
You don't really need this `newPreview` variable, nor the `addon` variable below - you could just inline these into the call to `render`.